ui/pages/_manufacturerKey/_fixtureKey.vue

```diff
@@ -344,16 +361,16 @@ export default {
       // adapted from https://stackoverflow.com/a/21553982/451391
       const match = url.match(/^.*?\/\/(?:([^:/?#]*)(?::([0-9]+))?)/);
       return match ? match[1] : url;
+    },
+    /**
+     * Format a date to display as a <time> HTML tag.
+     * @param {!Date} date The Date object to format.
+     * @returns {!string} The <time> HTML tag.
+     */
+    getDateHtml(date) {
+      return `<time datetime="${date.toISOString()}" title="${date.toISOString()}">${date.toISOString().replace(/T.*?$/, ``)}</time>`;
     }
   }
 };
 
-/**
- * Format a date to display as a <time> HTML tag.
- * @param {!Date} date The Date object to format.
- * @returns {!string} The <time> HTML tag.
- */
-function getDateHtml(date) {
```

Comment:
Why did you move this into the component's methods? It doesn't use `this`, so it should be outside.

Reply:
@FloEdelmann It's also used in the template above, so it needs to be in the component's scope.

Reply:
Ah okay, didn't catch that.